unifiednlp-base/src/main/java/org/microg/tools/selfcheck/NlpOsCompatChecks.java

```diff
@@ -74,17 +76,16 @@ private boolean checkSystemIsConfigured(Context context, ResultCollector collect
             }
         }
         collector.addResult(context.getString(R.string.self_check_name_nlp_package_name),
-                systemMatchesPackage ? Result.Positive : Result.Negative, context.getString(R.string.self_check_resolution_nlp_package_name));
-        
-        SharedPreferences mSharedPreferences = context.getSharedPreferences(Constants.APP_SETTINGS_NAME,
-                Context.MODE_PRIVATE);
-        SharedPreferences.Editor editor = mSharedPreferences.edit();
-        editor.putBoolean(Constants.OS_SUPPORTS_UNLP, systemMatchesPackage);
-        editor.apply();
-        
+                systemMatchesPackage ? Result.Positive : Result.Negative, context.getText(R.string.self_check_resolution_nlp_package_name));
```

Comment:
`getText()` gives a `CharSequence` (with optional formatting) whereas `getString()` gives a `String` (the required type here) - this causes a build failure

Reply:
@mar-v-in: I think it could be a good idea a "compile time" option to add logging functionality.